src/unicef_security/models.py

```diff
@@ -47,15 +50,17 @@ class Meta:
         swappable = 'BUSINESSAREA_MODEL'
 
 
-class User(AbstractUser, TimeStampedModel):
+class ExtendedAbstractUser(AbstractUser, TimeStampedModel):
```

Comment:
can't this be the swappable model?

Reply:
To my knowledge, you cannot have both abstract and swappable

Reply:
so swappable should be fine, no?

Reply:
yes :) since we're not going to use it on olc